Apps/Src/Task_PetWDog.c

```diff
@@ -30,7 +30,7 @@ void Task_PetWDog(void *p_arg) {
         OSMutexPost(&WDog_Mutex, OS_OPT_POST_NONE, &err);
         assertOSError(err);
 
-        OSTimeDly(20, OS_OPT_TIME_DLY, &err);
+        OSTimeDly(10, OS_OPT_TIME_DLY, &err);
         assertOSError(err);
```

Comment:
Was this decreased because we are not running the open wire checks anymore? Would there be anything wrong with leaving it at 200ms?

Reply:
I don't think it affects that much. 100 ms makes it use more CPU time, but makes us less likely to timeout the watchdog when we shouldn't